lib/package-card.coffee

```diff
@@ -68,6 +68,9 @@ class PackageCard extends View
     if atom.packages.isBundledPackage(@pack.name) and @type is 'theme'
       @statusIndicator.hide()
 
+    unless @hasSettings(@pack)
+      @settingsButton.hide()
```

Comment:
was just talking with @simurai and let's use `.remove()` here because it will help the button styles round corners correctly when one button is left.